vcpkg.json

```diff
@@ -5,6 +5,7 @@
     "sdl2-net",
     "openal-soft",
     "libsndfile",
+    "portmidi",
```

Comment:
This shouldn't be necessary anymore.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1635/commits/2a947a36ed4f34877186d75f3967e1f9518ea032